qualtran/bloqs/data_loading/select_swap_qrom.ipynb

```diff
@@ -0,0 +1,297 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b3668f15",
+   "metadata": {
+    "cq.autogen": "title_cell"
+   },
+   "source": [
+    "# SelectSwapQROM"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9cb4ef3e",
+   "metadata": {
+    "cq.autogen": "top_imports"
+   },
+   "outputs": [],
+   "source": [
+    "from qualtran import Bloq, CompositeBloq, BloqBuilder, Signature, Register\n",
+    "from qualtran import QBit, QInt, QUInt, QAny\n",
+    "from qualtran.drawing import show_bloq, show_call_graph, show_counts_sigma\n",
+    "from typing import *\n",
+    "import numpy as np\n",
+    "import sympy\n",
+    "import cirq"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f3785c3a",
+   "metadata": {
+    "cq.autogen": "SelectSwapQROM.bloq_doc.md"
+   },
+   "source": [
+    "## `SelectSwapQROM`\n",
+    "Gate to load data[l] in the target register when the selection register stores integer l.\n",
+    "\n",
+    "Let\n",
+    "    N:= Number of data elements to load.\n",
+    "    b:= Bit-length of the target register in which data elements should be loaded.\n",
+    "\n",
+    "The `SelectSwapQROM` is a hybrid of the following two existing primitives:\n",
+    "\n",
+    "    * Unary Iteration based `QROM` requires O(N) T-gates to load `N` data\n",
```

Comment:
This doesn't format well in the notebook.

Reply:
fixed